application.md

```diff
@@ -1,5 +1,141 @@
+their chances to be accepted into the OpenAstronomy GSoC programme.
+To ensure students interact with our communities before their applications are
+submitted, one of the requirements of applications will be that students must
+submit at least one patch as part of their application, which will necessarily
+require that the students interact with the existing developer community.
+Once we select students, we will encourage them to further discuss their project
+ideas with the community before the coding period starts in earnest.
+
+Each sub-organisation provides a number of communication channel to its users
+and developers.
+These include weekly Google hangouts, email lists, as well as an IRC chat room.
+We will expect the students to make use of all of these channels during their
+projects.
+We also will make specific requirements for the students which include weekly
+meetings with their mentors as well as weekly blog posts on the OpenAstronomy
+site describing their progress.
+They will also be invited to participate in the creation of posters,
+presentations and/or tutorials to show in upcoming scientific conferences.
+Students (as all contributors) will be required to write up a user guide and
+full documentation for their contribution.
+This will be reviewed by the community through the GitHub and BitBucket comment
+and annotation pull request platform.
+
+If the work done leads to a research publication, the student will be invited to
+collaborate in the preparation of the publication.
+These guidelines have already been successful applied as part of our previous
+participations in GSoC and the ESA Summer of Code programmes.
+
+[1] https://openastronomy.github.io/gsoc2015/student_guidelines.html
+
+
+## How will you keep students involved with your community after GSoC?
+
+We aim to involve all new people with our communities as much as we can.
+GSoC students will be encouraged no less.
+We hope to convince them of the attractive and usability that our projects have
+for the long term.
+As many solar and astrophysicists are new to Python, we will invite the students
+to give tutorials about their contribution to astronomy and astrophysics
+institutes near their home location.
+This will help the students get a better understanding of the needs in the
+community and to get closer to the users and the field.
+This may provide the student a big push in their career, and open new doors in
+the astronomical research world.
+Over the years various students have been giving talks at astronomical
+institutes in Austria, England, Ireland and India.
+
+To further encourage students, we will follow the example set by other GSoC
+projects (such as SymPy) and ask the students to spend a small fraction of their
+time reviewing others’ pull requests.
+We will ask them to help review one pull request per week, which should be no
+more than a 5% time commitment.
+
+## Has your org been accepted as a mentoring org in Google Summer of Code before?
+
+No
```

Comment:
I'm not sure if this is free-form on the submission form, but if it is it might be a good chance to say something like "Not the organization as whole.  But Astropy and Sunpy have been involved via umbrella organizations since 2013."